tests/pytest/test.py

```diff
@@ -301,6 +301,20 @@ def testMgetCommand(env):
     # Test that MGET fails on path errors
     r.expect('JSON.MGET', 'doc:0', 'doc:1', '42isnotapath').raiseError()
 
+def testToggleCommand(env):
+    """Test REJSON.TOGGLE command"""
+    r = env
+    r.assertOk(r.execute_command('JSON.SET', 'test', '.', '{"foo":true}'))
+    r.assertEqual(r.execute_command('JSON.TOGGLE','test','.foo'), 'false')
```

Comment:
I would add another toggle to 100% verify the alternation
` r.assertEqual(r.execute_command('JSON.TOGGLE','test','.foo'), 'true')`

Reply:
sure

Reply:
done